packages/eslint-config-react-native-community/package.json

```diff
@@ -23,7 +23,8 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.30.1",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-react-native": "^4.0.0"
+    "eslint-plugin-react-native": "^4.0.0",
+    "typescript": "^4.8.2"
```

Comment:
This package is used externally as well, but the `.eslintrc` which adds TS usage is internal to the repo. It would be ideal if the dependency is only added in the internal config.

Otherwise if used in the community ESLint config, it should maybe be a peerDependency with a loose constraint like eslint/prettier, so the app can control the version.

Reply:
Yes, I originally had it under `repo-config/package.json` but have this as currently trying to debug why danger isn't working. This didn't end up helping so will update this!